docs/streams/developer-guide/config-streams.html

```diff
@@ -677,6 +685,30 @@ <h4><a class="toc-backref" href="#id33">default.windowed.value.serde.inner</a><a
               <p>This is discussed in more detail in <a class="reference internal" href="datatypes.html#streams-developer-guide-serdes"><span class="std std-ref">Data types and serialization</span></a>.</p>
             </div></blockquote>
         </div>
+      <div class="section" id="rack-aware-assignment-tags">
+        <h4><a class="toc-backref" href="#id34">rack.aware.assignment.tags</a><a class="headerlink" href="#rack-aware-assignment-tags" title="Permalink to this headline"></a>
+        </h4>
+        <blockquote>
+          <div>
+            <p>
+              This configuration sets list of tag keys used to distribute standby replicas across Kafka Streams
+              instances. When configured, Kafka Streams will make a best-effort to distribute the standby tasks over
+              each client tag dimension.
+            </p>
+            <p>
+              Tags for the Kafka Streams instances can be set via <code class="docutils literal"><span class="pre">client.tag.</span></code>
+              prefix. Example:
+            </p>
```

Comment:
I think the doc would benefit from a more complete example. could you try to put the example from the KIP in  a concise way here?
```
# Kafka Streams Client 1
client.tag.zone: eu-central-1a
client.tag.cluster: k8s-cluster1
rack.aware.assignment.tags: zone,cluster
 
# Kafka Streams Client 2
client.tag.zone: eu-central-1b
client.tag.cluster: k8s-cluster1
rack.aware.assignment.tags: zone,cluster
 
# Kafka Streams Client 3
client.tag.zone: eu-central-1a
client.tag.cluster: k8s-cluster2
rack.aware.assignment.tags: zone,cluster
 
# Kafka Streams Client 4
client.tag.zone: eu-central-1b
client.tag.cluster: k8s-cluster2
rack.aware.assignment.tags: zone,cluster
```
Maybe it is possible to put the clients next to each other or two client in one row and the other two client below. Below the clients you could describe who get which standbys for the active tasks on each client.

Reply:
Done
![Screenshot 2022-03-28 at 15 41 14](https://user-images.githubusercontent.com/8927925/160400068-7ff79728-4555-4857-b8ee-0a7ea7aa7794.png)